src/icons/delete.tsx

```diff
@@ -1,10 +1,11 @@
 import React from "react";
 
 type DeleteIconProps = {
-  onClick: () => void;
+  onClick?: () => void;
+  style?: React.CSSProperties;
```

Comment:
Better than style, I think we have to create size props in the icon components like
`{ size = 20 }: { size?: number }`
and 
`     height={size}
      width={size}
`